package.json

```diff
@@ -25,7 +25,7 @@
   "license": "SEE LICENSE IN LICENSE.txt",
   "dependencies": {
     "portscanner": "^2.2.0",
-    "vscode-chrome-debug-core": "^6.7.8",
+    "vscode-chrome-debug-core": "^6.7.12",
```

Comment:
I just added a comment on the merge you did on core.. do take a look.

Reply:
It can be NaN only and it'll behave as we want it to in that case... Good catch! :)